docs/docs/get_started/kubernetes_tutorial.md

```diff
@@ -6,210 +6,229 @@ description: Secretless Broker Documentation
+1. Add that user's credentials to Kubernetes Secrets
+1. Configure Secretless to connect to PostgreSQL using those credentials
 
-## Sample Application
-
-The tutorial uses an existing [pet store demo application](https://github.com/conjurdemos/pet-store-demo) that exposes the following routes:
-
-- `GET /pets` to list all the pets in inventory
-- `POST /pet` to add a pet
-  - Requires **Content-Type: application/json** header and body that includes **name** data
-
-There are additional routes that are also available, but these are the two that we will focus on for this tutorial.
-
-Pet data is stored in a PostgreSQL database, and the application may be configured to connect to the database by setting the `DB_URL` environment variable in the application's environment (following [12-factor principles](https://12factor.net/)).
+**As the application developer:**
 
-See [Try The Running Application](#try-the-running-application) for examples of consuming the routes using `curl`.
+1. Configure the application to connect to PostgreSQL via Secretless
+1. Deploy the application and the Secretless sidecar
 
-The database is **credential-protected**. With the Secretless Broker, we will be able to set the `DB_URL` environment variable to a value of the form:  `postgresql://localhost:5432/${APPLICATION_DB_NAME}?sslmode=disable`. The application will not require knowledge of credentials to connect to the database.
+### Prerequisites
 
-Note that although the local application-to-Secretless connection does not require SSL, by default the Secretless-to-PostgreSQL connection uses `sslmode=require`. For more information on PostgreSQL SSL modes please see the
-[PostgreSQL documentation](https://www.postgresql.org/docs/9.6/libpq-ssl.html) or
-our documentation on the [PostgreSQL handler](/docs/reference/handlers/postgres.html).
+To run through this tutorial, you will need:
 
-## Steps for the admin user
++ A running Kubernetes cluster (you can use
+  [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) to run a
+  cluster locally)
++ [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) configured
+  to point to the cluster
++ [Docker CLI](https://docs.docker.com/install/)
 
-The following steps would be taken by an admin-level user, who has the ability to create and configure a database and to add secret values to a secret store.
+## Steps for Security Admin
 
-### Provision Target Services (optional)
+The Security Admin sets up PostgreSQL, configures Secretless, and has sole
+access to the credentials.
 
-If you already have a PostgreSQL server running and want to use that in this tutorial, please continue to [Setup And Configure PostgreSQL Target Service](#setup-and-configure-postgresql-target-service).
+### Create PostgreSQL Service in Kubernetes
 
-In this section, you will create the PostgreSQL storage backend in your Kubernetes cluster.
+If you already have PostgreSQL running and want to use your instance, jump
```

Comment:
does this add unnecessary complexity to the tutorial? is anyone going to want to muddy up their existing pg when we provide them a simple way to deploy a toy instance? I feel like we could drop this, and it would simplify a couple of other things below where we have to reference "if you're using your own DB ..."

Reply:
agreed.